presto-main/src/test/java/io/prestosql/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -103,6 +109,30 @@ public Object deserializeLongDecimalWithNull(LongDecimalWithNullBenchmarkData da
         return ImmutableList.copyOf(readPages(data.getPagesSerde(), new BasicSliceInput(data.getDataSource())));
     }
 
+    @Benchmark
+    public Object serializeInt96NoNull(LongTimestampBenchmarkData data)
```

Comment:
also add benchmarks for `Int128ArrayBlock`?

Reply:
LongDecimal benchmarks test Int128 block